subtitles/whisper_transcriber.py

```diff
@@ -1,6 +1,9 @@
 from transcriber import Transcriber
+from threading import Lock
 import whisper
 
+mutex_whisper = Lock()
```

Comment:
Since this is treated as a constant, I would suggest the following naming:
```suggestion
WHISPER_MUTEX = Lock()
```